src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,134 @@
+package wolox.training.models;
+
+import static wolox.training.utils.ErrorConstants.BOOK_ALREADY_OWNED;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity
+@Table(name = "t_user")
+public class User {
```

Comment:
When testing these enpoint with this many-to-many relationship, an error occurs when trying to display a book or a user since one contains the other and spring does not know where to stop this relationship. An infinite recursion would be generated.
To avoid this error, place this annotation on the class as follows:

```suggestion
@JsonIdentityInfo (generador = ObjectIdGenerators.PropertyGenerator.class, propiedad = "id")
public class User {
```


Place said annotation in the same way in the model book.